Dockerfile

```diff
@@ -123,18 +59,33 @@ RUN if [ ! -f /tmp/resources/jai-1_1_3-lib-linux-amd64.tar.gz ]; then \
     rm /tmp/jai-1_1_3-lib-linux-amd64.tar.gz && \
     rm -r /tmp/jai-1_1_3 && \
     rm /tmp/jai_imageio-1_1-lib-linux-amd64.tar.gz && \
-    rm -r /tmp/jai_imageio-1_1
+    rm -r /tmp/jai_imageio-1_1; \
+    fi
```

Comment:
we already do this. Is there any time we might need the JAI_IMAGEIO to be false. If not then the argument should not be used here